backtrace-library/src/main/cpp/backends/backend.cpp

```diff
@@ -0,0 +1,69 @@
+#include "backend.h"
+#include "client-side-unwinding.h"
+#include <android/log.h>
+#include <atomic>
+#include <mutex>
+
+#if CRASHPAD_BACKEND
```

Comment:
We should use `#ifdef X` or `#if defined(X)`

Reply:
Will change across files, thank you!